docs/source/getting-started/manual-download.rst

```diff
@@ -0,0 +1,42 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Manually installing the SDK
+***************************
+
+If you require a higher level of security, you can instead install the SDK by manually downloading the compressed tarball, verifying its signature, extracting it and manually running the install script.
+
+Note that the Windows installer is already signed (within the binary itself), and that signature is checked by Windows before starting it. Nevertheless, you can still follow the steps below to check its external signature file.
+
+To do that:
+
+1. Go to https://github.com/digital-asset/daml/releases. Confirm your browser sees a valid certificate for the github.com domain.
+2. Download the artifact (*Assets* section, after the release notes) for your platform as well as the corresponding signature file. For example, if you are on macOS and want to install release 0.13.27, you would download the files ``daml-sdk-0.13.27-macos.tar.gz`` and ``daml-sdk-0.13.27-macos.tar.gz.asc``. Note that for Windows you can choose between the tarball, which follows the same instructions as the Linux and macOS ones (but assumes you have a number of typical Unix tools installed), or the installer, which ends with ``.exe``. Regardless, the steps to verify the signature are the same.
```

Comment:
Can we replace this with the current version?

Reply:
Good point, will try `|release|` here too.

Reply:
So, as I suspected, `|release|` does not work in code blocks, nor (less expected) in inline code. I've updated all the references to `1.5.0` on this page for now.

Reply:
No, wait. Sample output doesn't work for a future version. 😞

I'll try to get GPG working again tomorrow. 🤷

Reply:
If we have to hardcode it anyway, I don’t care too much about the version so happy to leave it out for now.

Reply:
Updated to 1.4.0. There is actually no other change in the `gpg` output than the version number (in retrospect, I wish I'd looked at it closer before running through it again), but as it appears in multiple inline code and code blocks, it'd still be a pain to use the current version there.